lua/autorun/client/cl_cfc_performance.lua

```diff
@@ -25,63 +25,55 @@ local function removeHooks()
 end
 
 local function runCommands()
-    --RunConsoleCommand("cl_threaded_bone_setup", "1")
-    --RunConsoleCommand("cl_threaded_client_leaf_system", "1")
-    --RunConsoleCommand("gmod_mcore_test", "1") -- It was suggested to set this value to -2 for some reason
-    RunConsoleCommand("mat_queue_mode", "-1")
+    RunConsoleCommand("cl_threaded_bone_setup", "1")
+    RunConsoleCommand("cl_threaded_client_leaf_system", "1")
+    RunConsoleCommand("gmod_mcore_test", "1") -- It was suggested to set this value to -2 for some reason
+    RunConsoleCommand("mat_queue_mode", "-2")
     RunConsoleCommand("r_decals", "25") -- Max Decals
-    --RunConsoleCommand("r_queued_decals", "1")-- potentially unstable
-    --RunConsoleCommand("r_queued_ropes", "1") -- potentially unstable
+    RunConsoleCommand("r_queued_decals", "1")-- potentially unstable
+    RunConsoleCommand("r_queued_ropes", "1") -- potentially unstable
     RunConsoleCommand("r_queued_post_processing", "1")
     RunConsoleCommand("r_threaded_client_shadow_manager", "1")
     RunConsoleCommand("r_threaded_particles", "1")
     RunConsoleCommand("r_threaded_renderables", "1")
-    --RunConsoleCommand("r_queued_ropes", "1")
     RunConsoleCommand("studio_queue_mode", "1")
     RunConsoleCommand("prop_active_gib_limit", "0")
-    --RunConsoleCommand("snd_mix_async", "1") --Multithreaded sound. Causes the first few milliseconds of sounds to be cut off on some systems.
+    RunConsoleCommand("snd_mix_async", "1") --Multithreaded sound. Causes the first few milliseconds of sounds to be cut off on some systems.
 end
 
 local function disableWidgets()
 end
 
 local function enableEnhancements()
-    print( "[CFC Enhancements] Enabling CFC Performance Enhancements.." )
-
-    -- Safety timer. Lets the last couple of things load up before we run the enhancements.
-    timer.Simple(2, function()
-        removeHooks()
-        runCommands()
-        disableWidgets()
-        print( "[CFC Enhancements] CFC Performance Enhancements enabled! Enjoy enhanced performance!" )
-    end)
-end
-
-local function playerIsLoaded()
-    return IsValid(LocalPlayer())
+    removeHooks()
+    runCommands()
+    disableWidgets()
 end
 
-local function handleWaiterTimeout()
-    print( "[CFC Enhancements] CFC Waiter timed out! Enhancements won't be loaded." )
-end
+local function warnEnable( ply, message, source )
+    local log = source == "chat" and ply:ChatPrint or print
```

Comment:
```suggestion
    local log = source == "chat" and ply.ChatPrint or print
```
`:` is only for calling methods

Reply:
other than this the commands seem to work

Reply:
Good catch!